src/Handler/CommandNameExtractor/NamedCommandExtractor.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
```

Comment:
After some thought, rather than placing the NamedCommandExtractor and NamedCommand interface under the Handler namespace where they get a bit separated, it would be clearer to place them under a `Plugins\NamedCommand` namespace.

They'd still ship with the core packages but putting the two parts together under the same folder really improves the cohesion and the plugin aspect hints that they're an optional setup, rather than people assuming they need to implement a NamedCommand interface always.